monero/wallet_client.go

```diff
@@ -492,14 +493,14 @@ func (c *walletClient) generateFromKeys(
 
 	spendKey := "" // not used when only generating a view key
 	if sk != nil {
-		spendKey = sk.Hex()
+		spendKey = hex.EncodeToString(sk.Bytes())
```

Comment:
I kinda preferred the `Hex()` method, what do you think about having `Hex()` return no 0x-prefix and `String` returns with the prefix?